### ISSUE TITLE
Update training loss for train_helpful message

### ISSUE THREAD

**Issue by Pchatain** [scottviteri/MarkovianTraining#11]
Currently msg_loss calculates the loss on the content, not the message. This is correct for getting the 'rewards' but wrong for when updating weights of message sender.

https://github.com/scottviteri/CollaborativeTraining/blob/d5462d4474e6da75c7d663599a2a43b14a1b99ac/src/collaborative_experiments/train_helpful_message.py#L98

**Comment by Pchatain** [comment]
This line should instead feed into the model the prompt + content + message and only finetune on the best message.
